### ISSUE TITLE
False negative when using stylelint-enable on already enabled rule

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1917]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

When you use `stylelint-enable` on enabled rule twice.
Ref: https://github.com/stylelint/stylelint/pull/1874#r79310366

> Which rule, if any, is this issue related to?

No matter

> What CSS is needed to reproduce this issue?

e.g.

```css
/* stylelint-disable */
/* stylelint-enable bar */
/* stylelint-enable foo */
/* stylelint-disable foo */
/* stylelint-enable bar */
/* stylelint-enable */`
```

> What stylelint configuration is needed to reproduce this issue?

No matter

> Which version of stylelint are you using?

e.g. `7.3.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?
No matter

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No matter

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

Throw error `rule has already been enabled`

**Comment by evilebottnawi** [comment]
@davidtheclark @jeddy3 i think it is breaking changes and should have `major` milestone

**Comment by davidtheclark** [comment]
@evilebottnawi Eh ... I think it's a bug and should just be a patch.

**Comment by evilebottnawi** [comment]
@davidtheclark if some have this line in files, it is broke CI build

**Comment by davidtheclark** [comment]
@evilebottnawi Yes, but that will always happens if somebody has relied on a bug. This is clearly buggy behavior, I think, so if we get it to work properly, that's no different from fixing other bugs in semver. It might mean people have *fewer* errors reported — not sure.

Look at the new semver policy, and according to that this should be a "minor release": "A bug fix in a rule that results in stylelint reporting more errors." https://github.com/stylelint/stylelint/pull/1865/files#diff-04c6e90faac2675aa89e2176d2eec7d8R83

**Comment by evilebottnawi** [comment]
@davidtheclark Thanks for clarifying
